### PR TITLE
Add FedRAMP glossary entry with guidelines.

### PR DESCRIPTION
**Pull request by spank50**
Closes Issue #395
Updates added to f.adoc

### CODE REVIEW

**Comment by spank50** [comment]
Needs review and approval to merge

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc [hunk 1]
```diff
@@ -75,6 +75,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
 
 *See also*:
 
+//Add FedRAMP terminology Issue #395
+[[fedora-project]]
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc:79]: This needs to be updated to fedramp

```suggestion
[[fedramp]]
```

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc [hunk 2]
```diff
@@ -75,6 +75,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
...
+==== image:images/yes.png[yes] FedRAMP® (noun)
+*Description*: The Federal Risk and Authorization Management Program (FedRAMP®) is managed by the FedRAMP Program Management Office. Use the ® symbol at first usage in a document. For additional information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc:81]: Question: Should it be spelled out upon first use, or not? (Either way, I'd recommend mentioning that here)

A few suggestions too

```suggestion
*Description*: The _Federal Risk and Authorization Management Program (FedRAMP®)_ is managed by the FedRAMP Program Management Office. Use the registered trademark symbol (®) at first usage in a document. For more information, see the link:https://www.fedramp.gov/assets/resources/documents/FedRAMP_Branding_Guidance.pdf[FedRAMP Brand Guide].
```

**Comment by mportman12** [comment]
[PR 411](https://github.com/redhat-documentation/supplementary-style-guide/pull/411) seems to cover the same issue.

**Pull request closed without merging**